src/index.html

```diff
@@ -1,13 +1,353 @@
+              <h3 class="details__description-title">
+                Range & Integrated lights
+              </h3>
+              <div class="details__description-info">
+                The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </div>
+            </div>
+          </div>
+          <div class="details__photos">
+            <div class="grid grid--desktop grid--tablet grid--mobile">
+              <div class="grid__item grid__item1-4-d grid__item1-2-t">
+                <img
+                  src="./images/details/5.png"
+                  alt="Place 1"
+                  class="details__photo"
+                >
+              </div>
+              <div class="grid__item grid__item5-12-d grid__item3-6-t">
+                <img
+                  src="./images/details/6.png"
+                  alt="Place 1"
+                  class="details__photo"
+                  >
+              </div>
+            </div>
+            <div class="details__description">
+              <h3 class="details__description-title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+              <div class="details__description-info">
+                Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </div>
+            </div>
+          </div>
+
+          <div class="grid grid--desktop grid--tablet">
+            <div class="grid__item grid__item5-8-d grid__item2-5-t">
+              <a href="#" class="button">
+                Explore
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--desktop grid--tablet">
+            <div class="grid__item grid__item1-5-d grid__item1-3-t">
+              <label>
+                <form action="#">
+                  <input
+                  type="text"
+                  name="name"
+                  class="form-field contact-us__field"
+                  placeholder="Name"
+                  >
+                  <input
+                  type="email"
+                  name="email"
+                  class="form-field contact-us__field"
+                  placeholder="Email"
+                  >
+                  <textarea
+                    name="message"
+                    class="form-field form-field--textarea contact-us__field"
+                    placeholder="Message"
+                    >
+                  </textarea>
+                </form>
+              </label>
+              <button class="button">
```

Comment:
Does this button must to do smth on click?

Reply:
fixed.